layouts/index.html

```diff
@@ -27,14 +27,19 @@ <h3 class="subtitle is-3 {{ if .Site.Params.fadeIn | default true }}fade-in two{
     <div class="hero-foot {{ if .Site.Params.fadeIn | default true }}fade-in three{{ end }}">
     <hr>
     <nav class="nav-center">
-        <a class="nav-item" href="#about">About</a>
+        <a class="nav-item" href='{{ "/#about" | relLangURL }}'>{{ i18n "ABOUT_LINK" . }}</a>
```

Comment:
From my recollection, I think this should be  "#about" to make it work on sites with a sub-path in baseURL.

Reply:
Thanks, i will run some tests regarding this and check.